### DISCUSSION TITLE
Thank you Steve!

### DISCUSSION THREAD

**Discussion by ukw6** [omerfaruk-aran/esphome_samsung_hvac_bus#42] [General]
Just wanted to thank you, Steve for all the effort you put into this project. Look into your paypal account. ;) 
Although my old Samsung AC does not yet be completely compatible and the ESP still does send multiple diverting on/off and temperature signals, it is a start. And someday we won't need the cloud anymore.

**Comment by lanwin** [comment]
Wow! ❤️

**Reply by mbo18** [reply]
What’s your PayPal? Can’t find it thanks

**Reply by lanwin** [reply]
You can find it here paypal.me/lanwin

**Reply by mbo18** [reply]
Done! Thank you again

**Reply by lanwin** [reply]
:D
